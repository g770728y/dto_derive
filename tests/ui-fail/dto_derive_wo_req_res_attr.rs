extern crate dto_derive;
use dto_derive::Dto;

#[derive(Debug, PartialEq, Dto)] //~ ERROR required `request`/`response` attribute or struct name ending with `Request`/`Response`
#[dto(entity = "Entity")]
struct Dto { }

fn main() { }
