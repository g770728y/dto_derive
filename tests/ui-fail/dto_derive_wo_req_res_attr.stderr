error: required `request`/`response` attribute or struct name ending with `Request`/`Response`
 --> $DIR/dto_derive_wo_req_res_attr.rs:4:28
  |
4 | #[derive(Debug, PartialEq, Dto)] //~ ERROR required `request`/`response` attribute or struct name ending with `Request`/`Response`
  |                            ^^^

error: aborting due to previous error

